src/window/User Guide.md

```diff
@@ -0,0 +1,132 @@
+% User Guide to Using Piston Windows
+
+# Guide to using Piston Windows
+
+## Getting Help
+
+*Notice! Piston is being actively developed. Please let us know if anything in this tutorial is outdated.*
+
+See the main [Piston Guide](../../GUIDE.md) for general help with Piston.
+
+## Overview
+
+The piston engine is designed to be an abstraction. It provides the same functionality through several different backends, each of which have their own quirks and extra features.
+
+Because of this abstraction, some of the details of the piston API may seem unnecessary or useless. The purpose of this guide is to give concrete guidelines on the creation and usage of piston windows.
+
+If you mostly know what you're doing, but need a quick refresher, see the quick reference at the bottom of this guide.
+
+# Windows
+
+## Choosing a Backend
+
+There are a few available backends for piston, and it's important to choose one for your project.
```

Comment:
Maybe remove "available", since it might be misunderstood as there are some unavailable ones, developed in secret by people who don't want you to know what they are doing.

"it is"

"important to choose" Hmm.. this seems a bit urgent. The piston_window crate uses the Glutin backend by default, which people don't notice unless they change the generic parameter. In general, most people don't pick a window backend for their project, they do it when they ship the application. For example, to run a mobile device you might need another backend than the one you used to develop the app.